app/uk/gov/hmrc/transitmovementsrouter/models/errors/UpscanCallbackError.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.transitmovementsrouter.models.errors
+
+sealed trait UpscanCallbackError
+
+object UpscanCallbackError {
+  case object UnexpectedResponse extends UpscanCallbackError
```

Comment:
I can't see any usage of this Error also it is not mapped with ConvertError. if not using then please remove it.

Reply:
@drnaylor-oc  Created to be wrapped in an eitherT then realised we wouldn't be returning an error status code to upscan, I'll keep it for now in case my understanding is wrong

Reply:
As above. Note that because the trait is concerned with the presentation layer only, then we should just use a `PresentationError` directly, so this is still superfluous.